rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2248,8 +2273,39 @@ void rskTxWaitingForSignature_fail_adding_an_already_existence_key_after_rskip_3
         });
 
 
-        assertEquals(1, provider.getPegoutsWaitingForConfirmations().getEntries().size());
-        assertEquals(1, provider.getPegoutsWaitingForSignatures().size());
+        assertEquals(1, pegoutsWaitingForConfirmations.getEntries().size());
+        assertEquals(1, pegoutWaitingForSignatures.size());
```

Comment:
We could add an assertion that the migration tx is still waiting for signatures and the pegout is still waiting to be moved. i.e. that nothing changed